app/controllers/api/products_controller.rb

```diff
@@ -8,6 +8,42 @@ class ProductsController < Api::BaseController
 
     skip_authorization_check only: [:show, :bulk_products, :overridable]
 
+    def shop
+      renderer = ProductsRenderer.new(current_distributor, current_order_cycle)
+      products = ProductsFilterer.new(renderer.products).call
```

Comment:
The method on the renderer is `#products_json`

Reply:
it was json... now that we dont have cache I dont think we should waste time parsgin and unparsing json... some work to do here.